fileformats/ART_Reader.cpp

```diff
@@ -3,6 +3,7 @@
 #include "../ui/XL_Console.h"
 #include <string.h>
 #include <stdio.h>
+#include <stdlib.h>
```

Comment:
Why do we need stdslib? What error does this resolve?

Reply:
I do

```gcc -c *.cpp```

and get:

```
ART_Reader.cpp: In member function ‘virtual bool ART_Reader::OpenFile(const char*)’:
ART_Reader.cpp:109:45: error: ‘strtol’ was not declared in this scope
   s32 nIndex = (s32)strtol(pszFile, &tmp, 10);
```